server/webapp/WEB-INF/rails.new/lib/java_imports.rb

```diff
@@ -15,7 +15,6 @@
 ##########################GO-LICENSE-END##################################
 
 module JavaImports
-  java_import com.rits.cloning.Cloner unless defined? Cloner
```

Comment:
I think the Cloner is used is other places to clone entities or just parts of the config.